src/Pages/Authentication/Login.jsx

```diff
@@ -0,0 +1,92 @@
+import "./authentication.css";
+import { Link } from "react-router-dom";
+import { useState } from "react";
+import axios from "axios";
+import { useNavigate } from "react-router";
+import { useAuth } from "../../context/authentication-context";
+
+
+const Login = () => {
+
+    const navigate = useNavigate();
+    const [userData, setUserData] = useState({email : "", password : ""});
+    const { authDispatch } = useAuth();
+    console.log(userData);
+
+    // Dummy data for guest credential
+    const dummyData = {
+        email : "adarshbalika@gmail.com",
+        password : "adarshBalika123"
+    }
+
+    // Use guest credential click handler
+    const addDummyUser = (e) => {
+        e.preventDefault();
+        setUserData(dummyData);
+    }
+
+    // User input values change handler
+    const userInputValues = (e) => {
+        setUserData((pre) => ({...pre, [e.target.name] : e.target.value }));
+    }
+
+    // login click handler
+    const loginClickHandler = async (e) => {
+        e.preventDefault();
+        try {
+            const response = await axios.post("/api/auth/login", userData);
+            
+            localStorage.setItem("token", response.data.encodedToken);
+            authDispatch({ type : "USER_LOGIN", payload : response.data.encodedToken })
+            navigate("/");
+
+        } catch (error) {
```

Comment:
Catch Block only returns something if the ```response.status``` is 500.
We can check for other response status like user already exist using conditionals.